config/destination_test.go

```diff
@@ -0,0 +1,356 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/conduitio-labs/conduit-connector-gcp-pubsub/config/validator"
+	"github.com/conduitio-labs/conduit-connector-gcp-pubsub/models"
+)
+
+func TestParseDestination(t *testing.T) {
```

Comment:
If possible/practical, it would be good to see more tests related to opening the destination, writing records, handling errors there and similar.

Reply:
I've added the unit tests for interfaces' methods for source and destination:
Source:
- Next(ctx context.Context) (sdk.Record, error)
- Ack(context.Context) error
- Stop() error

Destination:
- Stop() error
(I haven't add a test for a `Publish(context.Context, sdk.Record, sdk.AckFunc)` method because it returns nothing).

[Commit e0dfa47](https://github.com/conduitio-labs/conduit-connector-gcp-pubsub/commit/e0dfa47ab9c19ab2b485a22c20769cf2800d8edc)